pkg/plugins/backendplugin/coreplugin/registry.go

```diff
@@ -145,6 +145,11 @@ func (l *logWrapper) Error(msg string, args ...interface{}) {
 	l.logger.Error(msg, args...)
 }
 
+// With creates a sub-logger that will always have the given key/value pairs.
+func (l *logWrapper) With(args ...interface{}) sdklog.Logger {
+	panic("TODO")
```

Comment:
See https://github.com/grafana/grafana-plugin-sdk-go/pull/578

@ferruvich any sense what implementation makes sense here?

Reply:
Hey @ryantxu 

On that PR I added some implementations examples that describe better how to construct `With`. It's basically a way to "persist" `args` in the logger so they are always logged, and not having to repeat them continuously on each log line.

Reply:
I've noticed that the `log.Logger` is implemented by a `ConcreteLogger` and a `Fake` logger (is there anything else I'm missing?).
 
You could extend the `log.Logger` interface with `With` there too, and then use the `gokitlog.With` in `ConcreteLogger`.